x/incentives/client/cli/query.go

```diff
@@ -104,6 +105,15 @@ func GetCmdUpcomingGaugesPerDenom() (*osmocli.QueryDescriptor, *types.UpcomingGa
 	}, &types.UpcomingGaugesPerDenomRequest{}
 }
 
+// GetCmdCurrentVolumeByGroupGaugeID returns current volume for each gauge respectively since the last epoch from a group gauge ID.
+func GetCmdCurrentVolumeByGroupGaugeID() (*osmocli.QueryDescriptor, *types.QueryCurrentVolumeByGroupGaugeIDRequest) {
```

Comment:
Let's call these weights. Volume is a specific implementation of group weights.

We should keep this more general

Reply:
Agreed, changed here https://github.com/osmosis-labs/osmosis/pull/6651/commits/c235e38c111f8a826263190419391341e5810e5f